docs/en/writing-running-appium/tutorial/swipe-tutorial.md

```diff
@@ -0,0 +1,41 @@
+## Scroll / Swipe actions tutorial
+
+### Android vs iOS UI differences
+
+Appium uses 'XCUITest' driver for iOS and 'UIAutomator2','Espresso' for
+Android. Default driver for Android is 'UIAutomator2'.
+
+Using default driver settings, we can see elements outside screen with
+iOS, but can't interact with them. We can check their values, but we
```

Comment:
XCUITest, actually, sometimes allows to tap an element, so then it would be automatically scrolled to the viewport. Although, it does not always work reliably.